### PR TITLE
 Task 143665 : Build and validate package 'upath'

### PR DESCRIPTION
**Pull request by vikasgupta8**
Created build script to build and validate the package **upath**.

### CODE REVIEW

File: u/upath/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
```
Comment (amitsadaphule) [inline u/upath/LICENSE:22]: The LICENSE file for all the build-scripts is supposed to be `Apache 2.0`. Please copy it from one of the other package's directory.

File: u/upath/upath_ubi_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=upath
+PACKAGE_VERSION=v2.0.1
+PACKAGE_URL=https://github.com/anodynos/upath
+
+yum -y update && yum install -y yum-utils nodejs nodejs-devel nodejs-packaging npm python38 python38-devel ncurses git gcc gcc-c++ libffi libffi-devel ncurses git jq make cmake
```
Comment (amitsadaphule) [inline u/upath/upath_ubi_8.4.sh:22]: Please avoid `yum update` unless absolutely needed for package dependencies.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

**Comment by amitsadaphule** [comment]
Also, in the future, please consider squashing commits before raising the PR.

File: u/upath/LICENSE (hunk 1 above)
Comment (vikasgupta8) [reply u/upath/LICENSE]: Replaced LICENSE file with Apache 2.0.

File: u/upath/upath_ubi_8.4.sh (hunk 2 above)
Comment (vikasgupta8) [reply u/upath/upath_ubi_8.4.sh]: Removed yum update.

**Review by amitsadaphule — APPROVED** [review]

**Pull request merged**